include/superior_mysqlpp/extras/member_wrapper.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+namespace SuperiorMySqlpp
+{
+    namespace detail
+    {
+        /* Wraps member function into functor */
```

Comment:
Even though this class usage is pretty obvious, please add standard documentation - for the class and also for members.
Also pls document `wrapMember` functions.